randovania/games/samus_returns/exporter/patch_data_factory.py

```diff
@@ -1,10 +1,146 @@
+import os
+
+from randovania.exporter import pickup_exporter
 from randovania.exporter.patch_data_factory import BasePatchDataFactory
+from randovania.exporter.pickup_exporter import ExportedPickupDetails
+from randovania.game_description.assignment import PickupTarget
+from randovania.game_description.resources.item_resource_info import ItemResourceInfo
+from randovania.game_description.resources.pickup_entry import ConditionalResources
+from randovania.game_description.resources.resource_info import ResourceCollection
+from randovania.game_description.world.node import Node
```

Comment:
All these imports of `randovania.game_description.world......` moved to `randovania.game_description.db....` 
Also`world` was renamed to `region`.
Please rename also the variables